ApplensBackend/Services/OutageCommunicationService/OutageCommunicationService.cs

```diff
@@ -0,0 +1,163 @@
+﻿using AppLensV3.Helpers;
+using AppLensV3.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AppLensV3.Services
```

Comment:
Currently this service in the controller is only used for outage queries. Shekhar do you think maybe in the future we can have a system level detector (just similar to monitoring and analytics system invoker, but it's for the whole system level), which will help us to run all the system-level queries? In that way, we don't need to add a service for specific purpose each time.

Reply:
As you see here, this service is part of Applens controller (not Diag Backend) We cannot create a detector or system invoker for this because we want the outage communication to show even when our backend is down or not reachable.